docs/fundamentals/toc.yml

```diff
@@ -227,6 +227,21 @@ items:
         href: ../core/tools/dotnet-msbuild.md
       - name: dotnet new
         href: ../core/tools/dotnet-new.md
+        items:
+        - name: dotnet new --list
+          href: ../core/tools/dotnet-new-list.md
+        - name: dotnet new --search
+          href: ../core/tools/dotnet-new-search.md
+        - name: dotnet new --install
+          href: ../core/tools/dotnet-new-install.md
+        - name: dotnet new --uninstall
+          href: ../core/tools/dotnet-new-uninstall.md
+        - name: dotnet new --update-*
+          href: ../core/tools/dotnet-new-update.md
+        - name: .NET default templates
+          href: ../core/tools/dotnet-new-sdk-templates.md
+        - name: Custom templates
+          href: ../core/tools/custom-templates.md
```

Comment:
We try to keep only one TOC entry per article, so it would be best to have this here or under Create templates for the CLI, but not in both places. I'm OK with moving it to here, so we just need to remove it from Create templates for the CLI

Reply:
Sounds good to me! Thank you all for cleaning this up!